app/models/post.rb

```diff
@@ -10,4 +10,27 @@ def comments_counter
   def likes_counter
     likes.count
   end
+
+  def update_posts_counter
+    author.update(posts_count: author.posts_count + 1)
+  end
+
+  def update_likes_counter
+    likes_count = likes.count
+    update(likes_count:)
+  end
+
+  def update_comments_counter
+    comments_count = comments.count
+    update(comments_count:)
+  end
+
+  def add_comment(comment)
+    comments << comment
+    update_comments_counter
+  end
```

Comment:
- [ ] [OPTIONAL] Instead of implementing a method here that creates a comment and uses `update_comments_counter`, please add the `update_comments_counter` method to the `Comment` model and use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comments_counter` method every time a new post has been saved to database.